src/main/java/org/isf/menu/gui/Menu.java

```diff
@@ -72,7 +71,6 @@ public static void checkJavaVersion() {
 	}
 
 	public static void main(String[] args) {
```

Comment:
Just from my ignorance about GUI project , why multiple `main` methods ? how are these used?

Reply:
My reading is that the multiple `main`s were used for testing purposes and as the testing matures they can be removed.

Reply:
well, "testing gui" is not only JUnit, but also "see" the resulting JFrame/JDialog and check the graphic layout... having the main with some mock data can speed up the thing

Reply:
OK, makes sense, if that is the point keep in mind that  is susceptible to automation too. I think lifetime for GUI project could be short at this point or maybe not, but I used [this](https://joel-costigliola.github.io/assertj/assertj-swing.html) for that kind of testing  some time ago!!!

Reply:
yes, but as you say the GUI project lifetime could make the effort worthless, thanks anyway to pointed it out!

Reply:
Note that OP-226 effort are headed to isolate and test the GUI logic only